src/parameterizations/vertical/MOM_internal_tide_input.F90

```diff
@@ -417,6 +426,8 @@ subroutine int_tide_input_init(Time, G, GV, US, param_file, diag, CS, itide)
 
   CS%diag => diag
 
+  W_m2_to_HZ2_T3 = GV%m_to_H*(US%m_to_Z**2)*(US%T_to_s**3)
```

Comment:
If the input units really are `W m-2`, we should have `W_m2_to_HZ2_T3 = GV%kg_m2_to_H*(US%m_to_Z**2)*(US%T_to_s**3)`.